doc/models/text_classification.rst

```diff
@@ -9,7 +9,7 @@ Input: Text sequence
 Output: Class label
 
 
-tree_lstm_model
+TreeLstmModel
```

Comment:
Hmm, you're right.  Really, it should just be `Lstm` everywhere, but I'll change this one to match for now, instead of changing everywhere else (including the code).